pkg/conversion/converter.go

```diff
@@ -213,6 +213,8 @@ type Meta struct {
 	// KeyNameMapping is an optional function which may map the listed key (field name)
 	// into a source and destination value.
 	KeyNameMapping FieldMappingFunc
+	// Context is an optional field that callers may use to pass info to conversion functions.
+	Context interface{}
```

Comment:
Conversion needs to be ignorant of who is using it I think.  And just
from a package structure point of view, I don't want conversion to
depend on runtime or an external package.  It truly is generic to how
it's used.